### DISCUSSION TITLE
验证黑号的方法有例外

### DISCUSSION THREAD

**Discussion by Boom-Hacker** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#243] [General]
我的号签到一天5分.申请正式版内测被拒,但是绑定miui手机显示20033,解锁工具显示账号受限

**Comment by xbn2002** [comment]
一共有三种黑号
绑定黑号：错误代码20032，账号权限不足或者账号受限
解锁黑号：错误代码20033，账号权限不足或者账号受限
小米社区黑号：一般来说是签到稳定+1，并且是**所有的项目**都不能过（正式版内测存在满员可能性，所以不太具有代表性）如果连小爱内测都过不去基本就无了

**Comment by xbn2002** [comment]
前两种黑号之间的联系是：当绑定黑号时，解锁也必然黑号；反之不然。
而小米社区黑号和前两者没有任何必然关系。

**Comment by Boom-Hacker** [comment]
![Screenshot_2024-04-14-09-46-13-113_com xiaomi vipaccount](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/95948654/24b8846d-0cde-4b2e-a7a4-16ad68904d1c)
我绑定解锁都黑，然后小米社区申请正式版内测因满员被拒，我周五晚上申请的小爱到现在也没过。但是我连续两天签到5个经验

**Comment by Boom-Hacker** [comment]
我现在是k40s已解锁，我给家人也叮嘱不要升级。这要真是个特例我可以给你们调查用我无所谓

**Comment by Boom-Hacker** [comment]
e,顺便一提，我记得这号是闲鱼买的，最开始的时候解锁没问题

**Comment by Boom-Hacker** [comment]
另外，就算我社区不是黑号，通过了bl解锁审核，那我解锁的时候难道能用另外一个账号？否则不还是不能解吗

**Comment by xbn2002** [comment]
> 另外，就算我社区不是黑号，通过了bl解锁审核，那我解锁的时候难道能用另外一个账号？否则不还是不能解吗

如果解锁或者绑定任一环节提示账号权限不足或者账号受限，就代表该号已黑，即使通过了澎湃BL锁审核，也不能解锁。

**Comment by Boom-Hacker** [comment]
> > 另外，就算我社区不是黑号，通过了bl解锁审核，那我解锁的时候难道能用另外一个账号？否则不还是不能解吗
> 
> 如果解锁或者绑定任一环节提示账号权限不足或者账号受限，就代表该号已黑，即使通过了澎湃BL锁审核，也不能解锁。

所以说，hyperos的解锁流程是什么？通过审核之后就能绑定然后走老路？

**Comment by Boom-Hacker** [comment]
顺便问下，我的k40s刚刚推送hyperos，我用卡刷刷上去会锁bl吗

**Comment by xbn2002** [comment]
> > > 另外，就算我社区不是黑号，通过了bl解锁审核，那我解锁的时候难道能用另外一个账号？否则不还是不能解吗
> > 
> > 
> > 如果解锁或者绑定任一环节提示账号权限不足或者账号受限，就代表该号已黑，即使通过了澎湃BL锁审核，也不能解锁。
> 
> 所以说，hyperos的解锁流程是什么？通过审核之后就能绑定然后走老路？

是的，本质上就是以前那一套，只是对版本号和机型加了额外校验而已，流程是完全一样的

**Comment by xbn2002** [comment]
> 顺便问下，我的k40s刚刚推送hyperos，我用卡刷刷上去会锁bl吗

不会，你怎么刷都不会，线刷只要你记得别选miflash的全部删除并lock，其余两个也都不会
另外就是别自己手痒去bootloader里面执行上锁命令就好了

**Comment by xbn2002** [comment]
> 我绑定解锁都黑，然后小米社区申请正式版内测因满员被拒，我周五晚上申请的小爱到现在也没过。但是我连续两天签到5个经验

绑定/解锁有一个受限，社区就没折腾的意义了，该换号换号吧

**Comment by Boom-Hacker** [comment]
> > 我绑定解锁都黑，然后小米社区申请正式版内测因满员被拒，我周五晚上申请的小爱到现在也没过。但是我连续两天签到5个经验
> 
> 绑定/解锁有一个受限，社区就没折腾的意义了，该换号换号吧

我小爱刚过了，这号是不是还能申请内测用
